elastic/kibana
# [Fleet] Refresh policies when deleting agent policy

## Summary

Resolve https://github.com/elastic/kibana/issues/172784 

Refresh policies by invalidating cache, when a policy is deleted.

## How to test?

Delete an agent policy for the policy page, you should see that policy removed from the list as soon it's deleted


https://github.com/elastic/kibana/assets/1336873/e7fdfeb1-0a90-41c4-b5d2-fe690ba94b3b
